### DISCUSSION TITLE
Ueberzugpp

### DISCUSSION THREAD

**Discussion by Euro20179** [pystardust/ytfzf#674] [General]
Continuing from #636

**Comment by jstkdng** [comment]
>I've implemented this, and in wezterm it just infinitely loads.

did you try master? iterm2 is not yet in a release

**Reply by Euro20179** [reply]
> did you try master? iterm2 is not yet in a release

Oops, tried it in master and it works, however it gets a little buggy, is there any way to work around this?
![temp](https://user-images.githubusercontent.com/56898436/235788539-f094a24a-12d6-4a78-b275-399b65603436.png)

edit:
this is not unique to iterm2, does this happen because of how i compiled it?

**Reply by jstkdng** [reply]
I also had those messages, they are libvips related. The issue was that libvips wasn't compiled in release mode. Are you using arch? if so, try switching to `libvips-git` in the AUR.

**Reply by Euro20179** [reply]
Yep that seems to fix it.

Also, iterm2 support is now in development branch

**Reply by jstkdng** [reply]
Nice, now some testing from actual iterm2 users could be nice. Hope you don't mind me commenting in some of your issues :)

**Reply by Euro20179** [reply]
Go ahead, finally being able to close #44 would be amazing.

**Reply by jstkdng** [reply]
I'll se what I can do
